### PR TITLE
feature/google-ads-api-default

### PR DESCRIPTION
**Pull request by fivetran-joemarkiewicz**
**Are you a current Fivetran customer?** 

Fivetran created PR

**What change(s) does this PR introduce?** 

The `api_source` variable is now defaulted to `google_ads` as opposed to `adwords`. The Adwords API has since been depricated by Google and is now no longer the standard API for the Google Ads connector. Please ensure you are using a Google Ads API version of the Fivetran connector before upgrading this package.
  - Please note, the `adwords` version of this package will be fully removed from the package in August of 2022 (when the V2 updates roll out).

**Did you update the CHANGELOG?** 

- [X] Yes

**Does this PR introduce a breaking change?**

- [X] Yes (please provide breaking change details below.)
- [ ] No  (please provide an explanation as to how the change is non-breaking below.)

The default nature of the `api_source` has changed. As such, we should treat this as a breaking change.

**Did you update the dbt_project.yml files with the version upgrade (please leverage standard semantic versioning)? (In both your main project and integration_tests)** 

- [X] Yes

**Is this PR in response to a previously created Bug or Feature Request**

- [X] Yes, Issue/Feature From The Ad Reporting Package [#51](https://github.com/fivetran/dbt_ad_reporting/issues/51)
- [ ] No 

**How did you test the PR changes?** 

 

- [X] CircleCi  
- [X] Local (please provide additional testing details below)

**Select which warehouse(s) were used to test the PR**
 

- [X] BigQuery
- [X] Redshift
- [X] Snowflake
- [X] Postgres
- [X] Databricks
- [ ] Other (provide details below)

**Provide an emoji that best describes your current mood**
 
🍎 

**Feedback**

We are so excited you decided to contribute to the Fivetran community dbt package! We continue to work to improve the packages and would greatly appreciate your [feedback](https://www.surveymonkey.com/r/DQ7K7WW) on our existing dbt packages or what you'd like to see next.

### CODE REVIEW

**Review by fivetran-sheringuyen — COMMENTED** [review]
Hey @fivetran-joemarkiewicz ! 

PR looks mostly good, just had a couple of comments that were similar to the source package.

I am also getting the same error I got running `dbt test` in the source package -- is it expected?
```
21:08:34  1 of 1 START test dbt_expectations_expect_column_values_to_not_match_regex_list_stg_google_ads__ad_history_source_final_urls__any___ [RUN]
21:08:36  1 of 1 WARN 1 dbt_expectations_expect_column_values_to_not_match_regex_list_stg_google_ads__ad_history_source_final_urls__any___ [WARN 1 in 1.67s]
21:08:36  
21:08:36  Finished running 1 test in 4.27s.
21:08:36  
21:08:36  Completed with 1 warning:
21:08:36  
21:08:36  Warning in test dbt_expectations_expect_column_values_to_not_match_regex_list_stg_google_ads__ad_history_source_final_urls__any___ (models/google_ads_connector/stg_google_ads.yml)
21:08:36    Got 1 result, configured to warn if != 0
21:08:36  
21:08:36    compiled SQL at target/compiled/google_ads_source/models/google_ads_connector/stg_google_ads.yml/dbt_expectations_expect_column_f02cb56d69a9df1e600b3958899ceaa5.sql
21:08:36  
21:08:36  Done. PASS=0 WARN=1 ERROR=0 SKIP=0 TOTAL=1
```

Additionally, I want to confirm that only `adwords` has the `*_click_performance` model.

OW once the tests pass (or confirmed that this is expected) then I'm happy to approve!

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# dbt_google_ads v0.7.0
+## 🚨 Breaking Changes 🚨
+- The `api_source` variable is now defaulted to `google_ads` as opposed to `adwords`. The Adwords API has since been depricated by Google and is now no longer the standard API for the Google Ads connector. Please ensure you are using a Google Ads API version of the Fivetran connector before upgrading this package. ([#32](https://github.com/fivetran/dbt_google_ads/pull/32))
```
[the remark this answers is not in the document]
Comment (fivetran-joemarkiewicz) [reply CHANGELOG.md:3]: 🤦 haha updated

**Comment by fivetran-joemarkiewicz** [comment]
Thanks for the review @fivetran-sheringuyen! I addressed your inline comments in the latest commit. Also, to answer your additional questions:

> I am also getting the same error I got running dbt test in the source package -- is it expected?

- Yes this is expected. This is a result of the `final_urls` field possibly containing multiple urls. In a similar fashion where we select the latest historical record for some history models, in this case we select only one url to include in the final model. This warning is by design to raise awareness to users that a url is being intentionally cut off. See [here](https://github.com/fivetran/dbt_google_ads_source/blob/425749a911a0c63701849e026bbe6ef613f308ac/models/google_ads_connector/stg_google_ads.yml#L59-L65) for the specific test that is providing the warning.

> Additionally, I want to confirm that only adwords has the *_click_performance model.

- Correct, only adwords leverages the `click_performance`. This is not supported or needed with the data we leverage from the `google_ads` version of the api.

**Review by fivetran-sheringuyen — APPROVED** [review]
Awesome, thanks for the updates @fivetran-joemarkiewicz, LGTM!

**Pull request merged**